setup.py

```diff
@@ -54,7 +54,7 @@ def _get_test_env():
 
     def run(self):
 
-        cmd = ['py.test']
+        cmd = ['py.test -vv']
```

Comment:
It's not that verbosity is bad but rather that this configuration should be done in `setup.cfg` so that it affects all runs of pytest, not just those through `setup.py`.